sdcm/cluster.py

```diff
@@ -1098,7 +1098,9 @@ def scylla_setup(self, disks):
         Setup scylla
         :param disks: list of disk names
         """
-        self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks {}'.format(','.join(disks)))
+        result = self.remoter.run('ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
```

Comment:
Is this command works on all supported distributions?
You could use "cut" instead of awk, it would be shorter... cut -d ":" -f 2

Reply:
`cut -d ":" -f 2` works, there is a redundant leading whitespace.

The code is from our artifact-test (https://github.com/scylladb/scylla-artifact-tests/blob/master/scylla-artifacts.py#L346), it works well with all distros.